app/main.py

```diff
@@ -2,0 +1,6 @@
+def copy_file(command: str) -> None:
+    command_list = command.split()
+    if len(command_list) == 3 and command_list[1] != command_list[2]:
+        with (open(command_list[1], "r") as main_file,
+              open(command_list[2], "w") as file_copy):
```

Comment:
It's a good practice to use descriptive and correct variable names. In this case, `main_file` and `file_copy` could be more descriptive, like `source_file` and `destination_file` respectively. This would make your code more readable and self-explanatory.